docs/wiki/quick_start.md

```diff
@@ -106,6 +106,27 @@ There are two options to be create a wallet, first is using the jsng shell where
 
 Now that the wallet is setup, you are ready for your first deployment.
 
+## Adding tokens to a stellar wallet
+To know how to obtain tokens [see here](https://github.com/threefoldfoundation/info_gridmanual/blob/development_capacitypool/src/2_gettingstarted/getting_started_get_tft.md)
+
+## Create your Capacity pool
+Before we get to deploying the actual Ubuntu container, we need to create a capacity pool which will be used by any solutions deployed.
+
+First step is to choose whether you want to create a new pool or extend an existing one.
+![](images/pool1.png)
+
+How much cu(cpu)/su(memory) your pool needs and the type of something you will pay for.
```

Comment:
@BolaNasr `something` is a place holder, I was asking what is meant by type?? and then u mention that instead of something

Reply:
and don't remove the question I was just writing the part that wold change from the sentence